hack/update-codegen.sh

```diff
@@ -72,10 +72,10 @@ chmod +x ${CODEGEN_PKG}/generate-internal-groups.sh
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 cd ${SCRIPT_ROOT}
-echo "Generating client,lister,informer for kubeflow.org/v1"
+echo "Generating client,lister,informer for kubeflow.org/v1 and kubeflow.org/v2alpha1"
 ${CODEGEN_PKG}/generate-groups.sh "client,lister,informer" \
```

Comment:
This script needs to be refactored to use [kube_codegen.sh ](https://github.com/kubernetes/code-generator/blob/master/kube_codegen.sh). The k8s deps need to be bumped to use the latest kube_codegen args. Will create a separate issue to track the same.

Reply:
Yes, that's right. After we upgrade the Kube library version from 1.29 to 1.30, we should do it.